NSS-Day-Cohort-53/tabloid-full-stack-always-polite-iguanas
# Bb displaying post details

#### Changes Made
1. Added file GetPostById method to API
2. Added `PostDetails.js` component to client
3. Modified file `ApplicationViews.js` to include routes for posts details
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout bb-DisplayingPostDetails
    ```
7. Open a new Terminal tab (⌘T) and navigate to the server directory.
8. Test app functionality.
    > When a logged in user clicks on the Posts link they are taken to a list of posts and each entry in the list has author name, title, content, and a details link. When the user clicks the details link they should be directed to a view displaying more details about that post
9. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.

Closes #43
